source/bellman_ford.h

```diff
@@ -16,7 +16,7 @@ vector<ll> bellman_ford(vector<array<int, 3>>& edges, int v, int src) {
 		}
 	}
 
-	for (int i = 0; i < v - 1 && relaxed; ++i) {
+	for (int i = 0; i < v && relaxed; ++i) {
```

Comment:
This changes the semantics from the "intuitive" one where a vertex is always at distance 0 from itself.

I'm not sure this is the right way - I'd rather put an explicit check in the task that you attempted than modify the general implementation, but I guess it can make sense in either way. If we go with this version, we should definitely put a note on the booklet clarifying this behaviour, or it's going to be quite confusing.

Reply:
Actually, nevermind, we already assign -INF to all vertices that are part of a cycle, so this change makes sense